src/components/Feed/Events/CollectionCreatedFeedEvent.tsx

```diff
@@ -0,0 +1,104 @@
+import breakpoints from 'components/core/breakpoints';
+import Button from 'components/core/Button/Button';
+import colors from 'components/core/colors';
+import InteractiveLink from 'components/core/InteractiveLink/InteractiveLink';
+import Spacer from 'components/core/Spacer/Spacer';
+import { BaseM, TitleXS } from 'components/core/Text/Text';
+import { useRouter } from 'next/router';
+import { useCallback, useMemo } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+import { pluralize } from 'utils/string';
+import { getTimeSince } from 'utils/time';
+import { CollectionCreatedFeedEventFragment$key } from '__generated__/CollectionCreatedFeedEventFragment.graphql';
+import FeedEventTokenPreviews, { TokenToPreview } from '../FeedEventTokenPreviews';
+import { StyledClickHandler, StyledEvent, StyledEventHeader, StyledTime } from './EventStyles';
+
+type Props = {
+  eventRef: CollectionCreatedFeedEventFragment$key;
+};
+
+export default function CollectionCreatedFeedEvent({ eventRef }: Props) {
+  const event = useFragment(
+    graphql`
+      fragment CollectionCreatedFeedEventFragment on CollectionCreatedFeedEventData {
+        eventTime
+        owner @required(action: THROW) {
+          username
+        }
+        collection @required(action: THROW) {
+          dbid
+          name
+          tokens @required(action: THROW) {
+            token {
+              dbid
+            }
+            ...EventMediaFragment
+          }
+        }
+      }
+    `,
+    eventRef
+  );
+  const { push } = useRouter();
+
+  const tokensToPreview = useMemo(() => {
+    return event.collection.tokens.filter((n) => !!n).slice(0, 4);
```

Comment:
nit: since this is a common thing when using graphql, there's an existing helper function called `removeNullValues`. can use `removeNullValues(event.collection.tokens)`

Reply:
brilliant

Reply:
swapped to removeNullValues